extensions/nimbus/token-validation/src/main/java/org/eclipse/dataspaceconnector/nimbus/token/NimbusValidationRule.java

```diff
@@ -0,0 +1,7 @@
+package org.eclipse.dataspaceconnector.nimbus.token;
+
+import com.nimbusds.jwt.JWTClaimsSet;
+import org.eclipse.dataspaceconnector.spi.iam.ValidationRule;
+
+public interface NimbusValidationRule extends ValidationRule<JWTClaimsSet> {
```

Comment:
I get the reason to add the `Nimbus` to the name of the interface, but I think it will make more sense to call this `JWTClaimValidationRule` or something like that.